project_priority_report/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Project Priority Report
+=======================
+
+This module extends the functionality of project report .
```

Comment:
This module extends the functionality of project adding a task report listed by priority.
You can use this report to communicate the priorities to the project members.